fetch.bs

```diff
@@ -589,6 +589,9 @@ user-agent-defined <a for=header>value</a> for the
 
  <li><p>A <dfn export for=body id=concept-body-total-bytes>total bytes</dfn> (an
  integer), initially 0.
+
+ <li><p>A <dfn export for=body id=concept-body-replayable>replayable flag</dfn> (a boolean),
+ initially false.
```

Comment:
Replaced boolean/true/false with (none)/set/unset.

We actually [clone a body](https://fetch.spec.whatwg.org/#concept-body-clone) regardless of the flag. So calling it cloneable  is confusing, I think.